lib/constants.ts

```diff
@@ -11,6 +12,8 @@ import {
 
 export const ZTG = 10 ** 10;
 
+export const MAX_IN_OUT_RATIO = new Decimal(1).div(3).toString();
```

Comment:
Did you consider get this from the chain config? Should be on the `swaps` property

Reply:
Was planning to, but Malte said it's going to be removed from the chain config in near future so I proceeded like this.

Reply:
Ah good to know